.github/CODEOWNERS

```
@@ -193,6 +193,10 @@
 # Design
 **/*.scss  @elastic/kibana-design
 
+# Enterprise Search
+/x-pack/plugins/enterprise_search/ @elastic/app-search-frontend @elastic/workplace-search-frontend
+/x-pack/test/functional_enterprise_search/ @elastic/app-search-frontend @elastic/workplace-search-frontend
+
```

Hey @constancecchen! Two things:

1. Would you be okay adding the below suggestion? On the Kibana side, the design team is responsible for reviewing Sass files to make sure everything meshes well with EUI and other plugins. Right now that's pretty much only Caroline Horn, but I'll be jumping in to help her with Sass reviews. This is what Caroline and I discussed as the next best step so that we can start having specific designers review specific files.
2. Would the following addition target all Sass files that will be used by Enterprise Search?

Let me know if you have any objections! Happy to chat about it more.

```suggestion
/x-pack/plugins/enterprise_search/**/*.scss @elastic/ent-search-design 

```

---

Oh sorry I just saw this! Yes no problem at all, merging this in now ✨